### ISSUE TITLE
Illuminance Map Scale Does Not Work

### ISSUE THREAD

**Issue by rpg777** [NREL/OpenStudio#2222]
Changing the max/min scale values merely changes the legend scale labels; the maps themselves are not, uh, re-mapped. See this [UH post](https://unmethours.com/question/17163/illuminance-map-scale-change-doesnt-work/) for details.

**Comment by DavidGoldwasser** [comment]
related to or duplicate of issue #1750

**Comment by rpg777** [comment]
Yes, same issue. Only with this ticket, the actual problem is described. Will add actual problem description to Issue #1750 and close this.

**Comment by evanweaver** [comment]
No longer supporting ResultsViewer.
